### ISSUE TITLE
New python breaks rivet

### ISSUE THREAD

**Issue by ktf** [alisw/alidist#560]
Running rivet-test seems to fail with:

```
+ rivet-mkanalysis ALICE_Test
Traceback (most recent call last):
  File "/mnt/mesos/sandbox/sandbox/sw/slc7_x86-64/Rivet/2.5.1-alice1-1/bin/rivet-mkanalysis", line 12, in <module>
    import rivet, sys, os
ImportError: No module named rivet
```

@dberzano @qgp any idea of what might be going wrong?

**Comment by dberzano** [comment]
Hmm might it be that `rivet` gets installed into the wrong place since now we have a more complicated `PYTHONPATH` than before? I have no time to look into it but it's a good idea to:

* see what's the value of `PYTHONPATH` before the script starts,
* see in the sandbox where/if the `rivet` module gets installed

**Comment by ktf** [comment]
The issue here was having `lib64` in place of `lib` as installation path for the python modules. This was fixed by #546.
